### DISCUSSION TITLE
Form success and error messages with half cache strategy

### DISCUSSION THREAD

**Discussion by dries-dekeersgieter** [statamic/cms#8781] [Troubleshooting]
I have a form with error and success messages like this:

```
{{ form:contact }}

        {{ if success }}
            ...
        {{ /if }}

        {{ if errors }}
            ...
        {{ /if }}
        
        ...
            
{{ /form:contact }}
```

I'm using the half cache strategy, but the contact URL is excluded in config/statamic/static_caching.php

`
    'exclude' => [
        "/contact",
    ]
`

For some reason, the error and success messages are not being displayed.
I've also tried wrapping them with the {nocache} tags, but that doesn't work either...

**Comment by jasonvarga** [comment]
Does it work with static caching disabled?

**Comment by dries-dekeersgieter** [comment]
With static caching disabled it works, yes.

**Comment by stefankempf** [comment]
I get the same issue. Disabled static caching, the success is showing. With half measure on, it isn't. Even if I wrap only the success tag or the whole form with a nocache tag.

**Comment by duncanmcclean** [comment]
There seems to be an open bug report for this issue: #7628
